actions/workflows/run_all_nightlies.yaml

```diff
@@ -9,16 +9,24 @@ input:
   - debug
 
 vars:
+# Can put back to just distros once U20 is released
   - distros:
       RHEL7: el7
       RHEL8: el8
       UBUNTU16: u16
       UBUNTU18: u18
+      UBUNTU20: u20
+  - stabledistros:
+      RHEL7: el7
+      RHEL8: el8
+      UBUNTU16: u16
+      UBUNTU18: u18
+
```

Comment:
```suggestion
      # UBUNTU20: u20
```

Reply:
You'll also need to revert where `ctx().stabledistros` is used back to just `ctx().distros`.

Reply:
So your suggestion of just keeping them separate and having it commented out is easier.